content/root/glossary.textile

```diff
@@ -0,0 +1,224 @@
+
+See the "batch request documentation":/rest-api/beta#batch for further information.
+
+h2(#capability). Capability
+
+A capability represents a collection of resources, each with a set of permitted operations. For example, a capability can grant permission to subscribe to the @finance:@ channel namespace. Capabilities are used to define the rights associated with an authentication credential such as an API key or token, although the rights expressed by a capability for a token cannot exceed those of its associated API key. 
+
+See the "capabilities documentation":/core-features/authentication/#capabilities-explained for further information.
+
+h2(#channels). Channels
+
+Channels are used to logically separate messages into different topics. Clients subscribe to channels for topics they are interested in, to receive all messages sent on those channels. When a publisher sends a message on a channel, all subscribers receive that message.
+
+See the "channels documentation":/core-features/channels for further information.
+
+h2(#channel-namespaces). Channel namespaces
+
+Channel namespaces provide a way for certain properties (such as persistence) to be assigned to a group of channels, rather than requiring them to be assigned to individual channels. The namespace is the first colon-delimited segment of a channel name, or the channel name itself if no colon exists. For example, the channels @finance@ and @finance:approvals@ would both be considered part of the @finance@ namespace.
+
+See the "channel namespace documentation":/realtime/channels#channel-namespaces for further information.
+
+h2(#connection). Connection
+
+A connection to Ably must be established and maintained for realtime communication. A connection goes through several different connection states throughout its lifecycle, such as @connecting@, @connected@ and @suspended@.
+
+See the "connection documentation":/realtime/connection for further information.
+
+h2(#dashboard). Dashboard
+
+The Ably dashboard is the user interface for managing applications, viewing statistics, creating API keys and setting up integrations and channel rules.
+
+Once you have "registered":https://ably.com/sign-up for an account, you can "sign in":https://ably.com/login to view your dashboard.     
+
+h2(#dead-letter). Dead letter queue
+
+Dead letter queues store messages that were intended to be delivered to a queue, but have failed to be processed correctly. They are created when a queue is provisioned, with one dead letter queue per application, not per queue.
+
+See the "dead letter queue documentation":/general/queues#deadletter for further information.
+
+h2(#delta-compression). Delta compression
+
+Delta compression is a feature that enables a subscriber to only receive the difference between the payload of a message and its predecessor. It is used when there is a degree of similarity between successive messages.
```

Comment:
I'm not sure 'predecessor' is the right word here? It implies that the later message overwrites the previous one (at least to my mind). If that's accurate, then great!

If not, perhaps "the difference between the payloads of the current message and the one sent before it" would be more accurate?

Reply:
This is a fair point and I'd rather be unambiguous - updated.